UnityProject/Assets/Scripts/Equipment/Equipment.cs

```diff
@@ -157,17 +157,13 @@ public IEnumerator SetPlayerLoadOuts()
 
             foreach (KeyValuePair<string, string> gearItem in gear)
             {
-				if (gearItem.Value.Contains("cloth"))
+				if (gearItem.Value.Contains(ClothFactory.ClothingHierIdentifier) ||
+					gearItem.Value.Contains(ClothFactory.HeadsetHierIdentifier))
 				{
 					GameObject obj = ClothFactory.Instance.CreateCloth(gearItem.Value, Vector3.zero);
 					ItemAttributes itemAtts = obj.GetComponent<ItemAttributes>();
 					SetItem(GetLoadOutEventName(gearItem.Key), itemAtts.gameObject);
-				} else if (gearItem.Value.Contains("headset")) {
-					GameObject obj = ClothFactory.Instance.CreateCloth(gearItem.Value, Vector3.zero);
-					ItemAttributes itemAtts = obj.GetComponent<ItemAttributes>();
-					SetItem(GetLoadOutEventName(gearItem.Key), itemAtts.gameObject);
-					obj.AddComponent<Headset>();
-				} else {
+				} else if (!String.IsNullOrEmpty(gearItem.Value)) {
 					Debug.Log(gearItem.Value + " creation not implemented yet.");
 				}
             }

```

Comment:
No else? Don't we like an error in this case, even if commented out?

Reply:
In that case it's trying to spawn an item for the empty hier field. Like glasses for a job that doesn't start with glasses. We don't need errors in that case. In fact, the correct approach would be to skip item spawning for those fields entirely. That would be part of the equipment.cs refactor I mentioned.

Reply:
Fair enough, and we wont want to throw out too much errors in that case.. it could clog the log without need to.